apps/events.md

```diff
@@ -17,6 +17,7 @@ These events are available:
 | `product` | immutable.js object | Is triggered when the customer selects a product. Informs about the respective product. |
 | `category` | immutable.js object | Is triggered when the customer selects a category. Informs about the respective category and related products.|
 | `cart:add` | object | Is triggered when the customer adds a product to the cart. Informs about the current state of the cart, e.g. included items. This event isn't triggered if the customer makes changes in the cart itself, e.g. changing the amount of an item. |
+| `order:completed` | object | Is triggered when the customer reaches the order confirmation page after completing a purchase. Gives further information on the order, e.g. on the billing address, the order number, and the selected shipping method. |
```

Comment:
Gives further information on the order, e.g. > Provides additional information about the order, such as